composer.json

```diff
@@ -42,13 +42,13 @@
     ],
     "phpstan-install": [
```

Comment:
Could you please synchronize it with contributte/bare repo? 

https://github.com/contributte/bare/blob/master/composer.json#L24-L27

Reply:
Whole installation process or just versions? :)

Reply:
Whole composer setup please.

Reply:
ok, synced Makefile/.travis.yml/composer scripts in https://github.com/contributte/social/pull/6/commits/5850f10dc53a235ecc49205a97fdec0eb723137a